docs/guides/cloud/branch-review.mdx

```diff
@@ -0,0 +1,150 @@
+
+To access Branch Review, navigate to the **Branches** tab in the left sidebar and select the branch you want to review. In order to see a comparison, the branch you select must have an open pull request.
+
+<DocsImage
+  src="/img/guides/cloud/branch-review/get-to-branch-review.jpg"
+  alt="Get to Branch Review"
+/>
+
+## Branch Details
+
+### View Options
+
+_todo: add note about setup/run recording which impacts default view_
+
+Branch Review is the default view of the **Branches** tab. You can also toggle to see a list of Runs filtered by this branch.
+
+<DocsImage
+  src="/img/guides/cloud/branch-review/branch-review-views.jpg"
+  alt="Displaying different views for branches"
+/>
+
+### Review Header
+
+The header includes the Git commit message, PR selector, PR status and base and target branch indicators with the Cypress Cloud test run ID number (#).
+
+If there are multiple pull requests open for the same branch, you can select the pull request you want to review from the dropdown to the right of the commit message.
+
+Clicking the branch indicators will link you directly to the appropriate [test overview](/guides/cloud/debugging/recorded-runs#Overview-tab) and hovering on these elements exposes additional test meta data.
+
+<DocsImage
+  src="/img/guides/cloud/branch-review/branch-review-header.jpg"
+  alt="Branch Review Header"
+/>
+
+### Review Test Status
+
+When on the review screen, you will see tabs Failures, Flaky, Pending, Added, and Modified. Each tab will show you the tests that fall into that category. You can click on a test to view the test details.
+
+<DocsImage
+  src="/img/guides/cloud/branch-review/branch-review-details2.jpg"
+  alt="Branch Review Details"
+  caption="Viewing new failing tests for this PR"
+/>
+
+:::info
+
+The status key in each of the tabs indicates:
+
+- <svg
+    width="41"
+    height="20"
+    viewBox="0 0 41 20"
+    fill="none"
+    xmlns="http://www.w3.org/2000/svg"
+  >
+    <path
+      d="M15 9L13 11L17 11L15 9Z"
+      fill="#C62B49"
+      stroke="#C62B49"
+      stroke-width="2"
+      stroke-linecap="round"
+      stroke-linejoin="round"
+    />
+    <path
+      d="M27.1221 15.1641C25.085 15.1641 23.7656 14.0293 23.6357 12.4707L23.6289 12.3887H24.8115L24.8184 12.457C24.9209 13.4277 25.8096 14.0977 27.1221 14.0977C28.4141 14.0977 29.2959 13.373 29.2959 12.3271V12.3135C29.2959 11.1035 28.4277 10.4268 27.0469 10.4268H25.8096V9.3877H26.9922C28.168 9.3877 28.9883 8.66309 28.9883 7.67871V7.66504C28.9883 6.63281 28.3115 6.03809 27.1084 6.03809C25.9463 6.03809 25.1191 6.65332 25.0166 7.65137L25.0098 7.71973H23.8271L23.834 7.65137C23.9912 6.04492 25.2627 4.97168 27.1084 4.97168C28.9883 4.97168 30.2256 6.01758 30.2256 7.51465V7.52832C30.2256 8.74512 29.3643 9.5791 28.1338 9.83887V9.86621C29.5693 9.98926 30.5469 10.8848 30.5469 12.2861V12.2998C30.5469 13.9473 29.0977 15.1641 27.1221 15.1641Z"
+      fill="#C62B49"
+    />
+  </svg>
+  total number increased in your branch
```

Comment:
maybe "added" or "introduced" or similar instead of "increased"? not sure what wording is the most clear

Reply:
Yeah we can tighten this up. This is largely the wording provided for the previous UI elements.